src/swish/Mf-a6le

```diff
@@ -8,28 +8,28 @@ ifneq (,$(shell "${CC}" --help=warnings 2> /dev/null | grep return-local-addr))
   HUSH:=${HUSH} -Wno-return-local-addr
 endif
 C = ${CC} -m64 -msse2 -fPIC -Wall -Wextra -Werror -O2 ${CPPFLAGS} ${CFLAGS} ${LDFLAGS}
-OsiObj=osi.o sha1.o sqlite.o sqlite3.o
+OsiObj=osi.o sha1.o sqlite.o sqlite3.o ${UvLib}/libuv_a.a
```

Comment:
This seems like a good move, but slightly changes the meaning of OsiObj across different Mf-a6* files.

Reply:
I pushed a commit that puts it back. Thanks!